test/index.spec.js

```diff
@@ -443,11 +443,11 @@ describe('helpers', () => {
   })
 
   describe('.nodeAddress', () => {
-    it('throws on non thinWaistAddress', () => {
+    it('throws on non invalid node address', () => {
```

Comment:
This is a double-negative, could be just `throws on valid node address` but the test title doesn't make sense. Either it's valid and doesn't throw, or it's invalid and throws.

Reply:
yes, my bad!